modules/xfeatures2d/doc/xfeatures2d.bib

```diff
@@ -1,89 +1,119 @@
+@incollection{Agrawal08,
```

Comment:
Could you please backport sorting onto 3.4 branch too?
With some note which order should be preserved (like "cite id" string)